ui/src/clientLibraries/components/ClientJavaOverlay.tsx

```diff
@@ -58,62 +58,37 @@ const ClientJavaOverlay: FunctionComponent<Props> = props => {
         template={initializeClientCodeSnippet}
         label="Java Code"
         defaults={{
-          server: 'serverUrl',
+          server: 'basepath',
```

Comment:
is there a way to put the actual URL in here? similar to how we do here? https://github.com/influxdata/influxdb/blob/master/ui/src/clientLibraries/components/ClientLibraries.tsx#L35

Reply:
nvm i think it does that already...